src/main.rs

```diff
@@ -0,0 +1,11 @@
+extern crate syn;
+
+use std::io::{self, Read};
+
+fn main() {
+    // let mut input = String::new();
+    // io::stdin().read_to_string(&mut input).unwrap();
+    //
+    // syn::parse_str::<syn::File>(&input).unwrap();
```

Comment:
Please remove this file, I don't think it's necessary.

Reply:
Weird that was not supposed to sneak in...